js/bitfinex.js

```diff
@@ -489,8 +489,20 @@ module.exports = class bitfinex extends Exchange {
         throw new NotSupported (this.id + ' ' + currency + ' not supported for withdrawal');
     }
 
-    async deposit (currency, params = {}) {
-        await this.loadMarkets ();
+    async createDepositAddress (currency, params = {}) {
+        let response = await this.fetchDepositAddress (currency, this.extend (
+            {'renew': 1},
+            params
+        ));
```

Comment:
Can you, plz, put this line in either of the following ways:

```
let response = await this.fetchDepositAddress (currency, this.extend ({
    'renew': 1,
}, params));
```

or

```
let response = await this.fetchDepositAddress (currency, this.extend ({ 'renew': 1 }, params));
```

Otherwise our very sensitive transpiler complains. We know it can be frustrating sometimes, but this is currently our best options. Thx for your understanding!

Reply:
Sure. I checked the Travis log, but it complained about `wex`.

Reply:
Oh, sorry, anyway, it must have worked by chance, we would better keep it uniform. I edited the YoBit PR for this as well. Thx!